samples/benchmarks/coremark/boards/nrf54lm20pdk_nrf54lm20a_cpuapp.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include "app_aliases_common.overlay"
+
+/* The following configuration is required to run the CPUFLPR core.
+ * It is imported from the nordic-flpr snippet.
```

Comment:
```suggestion
 * It is imported from the nordic-flpr snippet in the sdk-nrf repository.
```

Verify if this configuration is equal to the following snippet

https://github.com/nrfconnect/sdk-nrf/blob/main/snippets/nordic-flpr/soc/nrf54lm20a_cpuapp.overlay